holidays/countries/sweden.py

```
@@ -88,39 +108,89 @@ def _populate_public_holidays(self):
         self._add_christmas_day_two(tr("Annandag jul"))
 
-        # New Year's Eve.
-        self._add_new_years_eve(tr("Nyårsafton"))
-
         # Optionally add all Sundays of the year.
         if self.include_sundays:
             for dt in _get_all_sundays(self._year):
                 # Sunday.
                 self._add_holiday(tr("Söndag"), dt)
 
+    def _populate_common(self):
```

Ruff (0.14.6)

Missing return type annotation for private function `_populate_common`

Add return type annotation: `None`

(ANN202)